### DISCUSSION TITLE
Top Langs - Private Commits

### DISCUSSION THREAD

**Discussion by prisnguyen** [anuraghazra/github-readme-stats#3515] [Q&A] [answered]
I'm curious whether the top-langs link showcases stats for private commits and repositories. If it does, could you kindly guide me on the specific parameter to use for this?

Thank you!

**Comment by qwerty541** [comment, marked as answer]
Hey, Priscilla! By default both stats and top langs card shows only your public contributions. To display private contributions you should [deploy own Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using your own GitHub API token with access to private repositories.
